api/src/main/scala/dx/api/DxApplet.scala

```diff
@@ -22,24 +22,26 @@ case class DxAppletDescribe(project: String,
                             runSpec: Option[JsValue] = None,
                             access: Option[JsValue] = None,
                             ignoreReuse: Option[Boolean] = None,
-                            hidden: Option[Boolean] = None)
+                            hidden: Option[Boolean] = None,
+                            treeTurnaroundTimeThreshold: Option[Long] = None)
```

Comment:
Can you add a docstring or comment on what this input is?

Reply:
Here and in other places where needed.

Reply:
Added in [2072d6d](https://github.com/dnanexus/dxScala/pull/123/commits/2072d6debfbbfbc12d29e76767905406f4145daf)